<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главное окно</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="treeView"/>
       <widget class="TreePropView" name="treeViewObjAttr"/>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="treeViewType"/>
       <widget class="QTreeView" name="treeViewLink"/>
       <widget class="QTreeView" name="treeViewStat"/>
       <widget class="QTreeView" name="treeViewAttr"/>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pshBtnTestProject">
        <property name="text">
         <string>Тест проекта</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnModelTest">
        <property name="text">
         <string>Тест модели</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnTypeTest">
        <property name="text">
         <string>Тест типов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnLinkTest">
        <property name="text">
         <string>Тест связей</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnStatTest">
        <property name="text">
         <string>Тест состояний</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pshBtnAttrTest">
        <property name="text">
         <string>Тест атрибутов</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblIcon">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>171</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TreePropView</class>
   <extends>QTreeView</extends>
   <header>treepropview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
